bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/troubleshoot/StrandDump.java

```diff
@@ -18,59 +18,128 @@
 
 package io.ballerina.runtime.internal.troubleshoot;
 
-import io.ballerina.runtime.internal.scheduling.Strand;
+import com.sun.management.HotSpotDiagnosticMXBean;
 
+import java.io.File;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import javax.management.MBeanServer;
+
 
 /**
  * Used to get the status of current Ballerina strands.
  *
  * @since 2201.2.0
  */
 public class StrandDump {
+    private static final String hotSpotBeanName = "com.sun.management:type=HotSpotDiagnostic";
```

Comment:
We need to use capital letters for java constants var names

Reply:
Fixed in c2a6a145269ecee260de8347a1fffadd28d287ed